apps/app/src/hooks/usePermissions.ts

```diff
@@ -0,0 +1,37 @@
+import { usePhoton } from '@photonhealth/react';
+import jwtDecode from 'jwt-decode';
+import { Permission } from 'packages/sdk/dist/types';
+import { useEffect, useState } from 'react';
+
+function checkHasPermission(subset: Permission[], superset: Permission[]) {
```

Comment:
is this the same idea?

`return subset.every(permission => superset.includes(permission));`

Reply:
love it